src/view/com/util/post-embeds/VideoEmbedInner/VideoEmbedInnerWeb.tsx

```diff
@@ -33,47 +34,57 @@ export function VideoEmbedInnerWeb({
   const hlsRef = useRef<Hls | undefined>(undefined)
 
   useEffect(() => {
-    if (!ref.current) return
-    if (!Hls.isSupported()) throw new HLSUnsupportedError()
+    let hls: Hls | undefined
+    // @ts-expect-error - dynamic import
+    import('hls.js/dist/hls.min').then(
+      ({default: DynamicHls}: {default: typeof Hls}) => {
+        setHlsLoading(false)
```

Comment:
you probably want to exit here if cleanup has already ran (e.g. if `embed.playlist` changes faster than `Hls` loads